docs/source/developer/interactives.rst

```diff
@@ -112,32 +49,52 @@ Every interactive will follow a very similar file structure as given below:
       ├── static/
       │   ├── interactives/
       │   │   └── <interactive-slug>/
-      │   │       ├── css/
+      │   │       ├── scss/
       │   │       │   ├── third-party/
-      │   │       │   └── style.css
+      │   │       │   └── <interactive-slug>.scss
       │   │       ├── img/
       │   │       ├── js/
       │   │       │   ├── third-party/
-      │   │       │   └── script.js
-      │   │       ├── README.md
-      │   │       └── thumbnail.png
+      │   │       │   └── <interactive-slug>.js
+      │   │       ├── package.json
+      │   │       └── README.md
       │   ├── js/
       │   │   └── third-party/
-      │   └── css/
+      │   └── scss/
       │       └── third-party/
       └── templates/
           └── interactives/
-              └── <interactive-template>.html
+              └── <interactive-slug>.html
 
 .. note::
 
   Each interactive has a unique ``slug``, which is used in file/directory names and urls. For example the slug for the ``High Score Boxes`` interactive is ``high-score-boxes``.
 
 Items of interest from this diagram:
 
-- ``static/interactives/<interactive-slug>/`` - This is the directory where static files for the interactive are stored, which should be broken down into ``css``, ``js`` and ``img`` directories.
+- ``static/interactives/<interactive-slug>/`` - This is the directory where static files for the interactive are stored, which should be broken down into ``scss``, ``js`` and ``img`` directories.
+
+- ``static/interactives/<interactive-slug>/package.json`` - The ``package.json`` file is npm's configuration file for projects and modules.
+  Here we include any dependencies the interacive has.
+  This file should only be included if the interactive loads **at least 1** npm module.
+  An example structure of a package.json file for an interactive is below:
 
-- ``static/interactives/<interactive-slug>/js/third-party/`` - Any third party JavaScript libraries specific to this interactive must be stored in this directory (unless you are using a CDN)
+  .. code-block:: none
+
+    {
+      "name": "csfg-interactive-<interactive-slug>",
+      "version": "1.0.0",
+      "private": true,
+      "dependencies": {
+          "<module-name>": "<module-version>"
+      }
+    }
+  
+  You will need to make sure the path to this file is listed in the ``csfieldguide/package.json`` file.
+  More information on npm modules can be found on the :ref:`static files` page.
+
+- ``static/interactives/<interactive-slug>/js/third-party/`` - Third-party JavaScript libraries specific to this interactive should be loaded as modules through the ``package.json`` file (see above).
+  Add the source JS file to this directory only if that is not possible (for example, if the package is not available on npm).
 
 - ``third-party/`` - Any third party css or js libraries used in multiple interactives can be stored in ``static/css/third-party/`` and ``static/js/third-party/`` directories.
```

Comment:
Perhaps add 'again, only if it is not possible to load the module through package.json'